llvm/lib/Transforms/Utils/LoopSimplify.cpp

```diff
@@ -777,8 +777,8 @@ INITIALIZE_PASS_BEGIN(LoopSimplify, "loop-simplify",
 INITIALIZE_PASS_DEPENDENCY(AssumptionCacheTracker)
 INITIALIZE_PASS_DEPENDENCY(DominatorTreeWrapperPass)
 INITIALIZE_PASS_DEPENDENCY(LoopInfoWrapperPass)
-INITIALIZE_PASS_END(LoopSimplify, "loop-simplify",
```

Comment:
I think loop-simplify is a transformation pass, not an analysis?

Reply:
Oh yes it's not an analysis. I had naively logged a list of passes being `addRequired` that weren't marked as analyses.
Reverting this line.